src/cascadia/TerminalSettingsModel/enableColorSelection.json

```diff
@@ -364,7 +403,8 @@
                 "foreground": "i00",
                 "background": "i03"
             },
-            "keys": "ctrl+shift+0"
+            "keys": "ctrl+shift+0",

```

Comment:
Hey, does having `keys`, `command` and `id` in the same settings file -- even though it is a SYSTEM settings file -- result in us changing the user's `keys` list? (e.g. do we accidentally up-convert these key/command/id triples into key/id and action/id pairs?)

Reply:
We do convert these into key/id and action/id pairs in our internal maps, but they're still in the `InBox` layer so they don't show up in the user's settings file